setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = bosi
-version = 0.0.177
+version = 0.0.179
```

Comment:
master branch is usually 0.0.x - basically some random number always increasing.
once we cut/branch-off for 4.0, this version is set to 4.0.x

that's what you'll find in 3.7 and 3.6.